### DISCUSSION TITLE
Add owc startup animation to intros.

### DISCUSSION THREAD

**Discussion by PMorghK** [ppy/osu#15722] [Ideas]
title says it all. also Hi.

**Comment by smoogipoo** [comment]
I don't actually understand what you want. Which startup animation (the countdown?) and which intros (the osu!-lazer startup ones?)?

**Reply by PMorghK** [reply]
Yes actually after the countdown ends theres a sort of startup in owc. I want that when I start lazer.

**Reply by PMorghK** [reply]
https://user-images.githubusercontent.com/43874888/142804460-1a3598a0-bf54-4f1e-b338-58354969806b.mp4
This right here.

**Reply by Loreos7** [reply]
As I understand it, the user suggests adding "seasonal intro sequences" when tournaments take a place. Sounds great.
Think its better to implement "always/sometimes/never show seasonal intro sequences".
